### ISSUE TITLE
Authentication failing on both user and admin types

### ISSUE THREAD

**Issue by bananasov** [sreedevk/pocketbase-sdk-rust#12]
Heya, i am trying to authenticate as an admin user but it keeps on erroring. I have no clue if its an issue with my code, pocketbase or this crate. I am running pocketbase version 0.10.2.

My code:
```rs
let mut client = Client::new("http://localhost:8090/api/").unwrap();
let auth = client.auth_via_email(
       String::from("test@test.com"),
       String::from("1234567890"),
       UserTypes::Admin /* use UserTypes::Admin for admin Authentication */
).await;
assert!(auth.is_ok())
```

**Comment by bananasov** [comment]
Just tried testing the pocketbase JS SDK and it worked fine with authentication.

**Comment by sreedevk** [comment]
@Autist69420 Looking into this.

**Comment by bananasov** [comment]
Alrightly! let me know if its resolved. don't overwork yourself

**Comment by sreedevk** [comment]
@Autist69420 Thank you for bringing up this issue. While debugging, I noticed that the authentication URLs in the latest pocketbase version had changed. I've made the necessary changes in [0.0.7](https://github.com/sreedevk/pocketbase-sdk-rust/releases/tag/0.0.7)

There maybe other URL changes as well... I am taking a look at them and will update any other pocketbase API URLs if required and push a new release. Once again, Thank you for your contribution.
